src/frequenz/sdk/timeseries/ev_charger_pool/_state_tracker.py

```diff
@@ -59,6 +59,14 @@ def from_ev_charger_data(cls, data: EVChargerData) -> EVChargerState:
             return EVChargerState.EV_PLUGGED
         return EVChargerState.IDLE
 
+    def ev_connected(self) -> bool:
+        """Check whether an EV is connected to the charger.
+
+        Returns:
+            Whether an EV is connected to the charger.
```

Comment:
What about something around `True whether an EV is connected to the charger, False otherwise` instead?

Reply:
ok, I didn't know about that pattern. However I have to admit this sounds to me like returning a value is optional rather than a boolean flag is returned, but this is probably just me :D

+1 about renaming the function to `is_ev_connected()`

Reply:
renamed.